docs/documented_modules.txt

```diff
@@ -34,6 +34,7 @@ open3d.pipelines.color_map
 open3d.pipelines.integration
 open3d.pipelines.odometry
 open3d.pipelines.registration
+open3d.t.pipelines.registration
```

Comment:
I believe we need `open3d.t.pipelines`, and for consistency, sort the list such that this appears next to other `open3d.t` items.

Reply:
Currently it is not showing up:
![image](https://user-images.githubusercontent.com/1501945/114261934-f84c7500-9991-11eb-8593-011953e7a37f.png)

Reply:
Resolved.